shared/util/kbfs.js

```diff
@@ -32,3 +37,69 @@ export function cleanup (folderName: string): string {
 
   return folderName.replace(/\s/g, '').replace(/\.\./g, '').replace(/\//g, '').replace(/\\/g, '')
 }
+
+export function decodeKBFSError (user: string, notification: FSNotification): Array<string> {
```

Comment:
kinda prefer this to return an object vs an array, {title, message} seems more clear than [0, 1]